tests/Scripting/TaskProperties.test.ts

```diff
@@ -34,6 +34,15 @@ describe('task', () => {
         markdownTable.verifyForDocs();
     }
 
+    beforeAll(() => {
+        jest.useFakeTimers();
+        jest.setSystemTime(new Date(2023, 6 - 1, 12));
```

Comment:
Thanks for fixing this. I could have sworn that I had found it and fixed it... 

Could the date be passed in as a string please? It turns out that this works perfectly well and is a lot more readable as it not prone to the off-by-one errors made likely with the month number when integers are passed in.

Reply:
> Could the date be passed in as a string please? It turns out that this works perfectly well and is a lot more readable as it not prone to the off-by-one errors made likely with the month number when integers are passed in.

oh, that's better indeed =)

Reply:
done =)

Reply:
Thanks!